notify/receivers.go

```diff
@@ -347,8 +347,12 @@ type NotifierConfig[T interface{}] struct {
 	Settings T
 }
 
+// GetDecryptedValueFn is a function that returns the decrypted value of
+// the given key. If the key is not present, then it returns the fallback value.
+type GetDecryptedValueFn func(ctx context.Context, sjd map[string][]byte, key string, fallback string) string
+
```

Comment:
This function is moved from the receivers package because it is not used there anymore.